contrib/upload_docs.sh

```diff
@@ -15,6 +15,7 @@ git remote show origin &>/dev/null || git remote add origin https://github.com/o
 git fetch --all
 git checkout -t origin/master -f
 git pull
+git submodule update --init --recursive
```

Comment:
why it is needed?

Reply:
@mike-dubman so that the documentation will be generated including UCG material

Reply:
call should be wrapped with `--with-ucg`

Reply:
@mike-dubman you're right, fixed.